stdpopsim/slim_engine.py

```diff
@@ -848,9 +848,16 @@ def matrix2str(
 
     # Mutation types.
     for i, m in enumerate(mutation_types):
-        distrib_args = [str(arg) for arg in m.distribution_args]
-        distrib_args[m.Q_scaled_index] = "Q * " + distrib_args[m.Q_scaled_index]
-        distrib_args = ", ".join(distrib_args)
+        if len(m.Q_scaled_index) >= 1:
+            distrib_args = [str(arg) for arg in m.distribution_args]
+            for j in m.Q_scaled_index:
+                distrib_args[m.Q_scaled_index[j]] = (
+                    "Q * " + distrib_args[m.Q_scaled_index[j]]
+                )
+            distrib_args = ", ".join(distrib_args)
+        # dealing with distributions given by "s" Eidos script:
+        else:
+            distrib_args = m.distribution_args
```

Comment:
Codecov is claiming that this line isn't covered by the tests. Looking at your tests, I don't understand why that would be the case. Maybe someone else has some insight? @mufernando ?

Reply:
Oh, I see now. The tests are creating the "l" MutationType, but there are no tests that run a simulation with this. @izabelcavassim could you please add a test that does a `dry_run` simulation? There are a couple of examples of this type of test in the test_slim_engine.py file.

Reply:
it looks likle this was fixed. correct @izabelcavassim ?